api_spec.yaml

```diff
@@ -4413,87 +4261,995 @@ paths:
+        Get a tenant config of the tenant specified by the url prefix.
+
+        SuperTokens subscription license key is required if querying any tenant other than the base tenant.
+
+        Returns recipes with their enabled flag and recipe specific configs saved in core
+
+      operationId: tenantConfigGet
+      parameters:
+        - $ref: '#/components/parameters/multitenancyRid'
+        - $ref: '#/components/parameters/api-key'
+        - $ref: '#/components/parameters/cdi-version'
+      responses:
+        200:
+          description: Tenant config result
+          content:
+            application/json:
+              schema:
+                oneOf:
+                - type: object
+                  required:
+                    - status
+                    - emailPassword
+                    - thirdParty
+                    - passwordless
+                  properties:
+                    status:
+                      $ref: '#/components/schemas/statusOK'
+                    emailPassword:
+                      type: object
+                      required:
+                        - enabled
+                      properties:
+                        enabled:
+                          type: boolean
+                    thirdParty:
+                      type: object
+                      required:
+                        - enabled
+                      properties:
+                        enabled:
+                          type: boolean
+                        providers:
+                          type: array
+                          items:
+                            $ref: '#/components/schemas/thirdPartyProviderConfig'
+                    passwordless:
+                      type: object
+                      required:
+                        - enabled
+                      properties:
+                        enabled:
+                          type: boolean
+
+        '400':
+          $ref: '#/components/responses/400'
+        
+        '401':
+          $ref: '#/components/responses/401'
+        
+        '402':
+          $ref: '#/components/responses/402'
+
+        '404':
+          $ref: '#/components/responses/404'
+        
+        '500':
+          $ref: '#/components/responses/500'
+
+  /appid-<appId>/recipe/multitenancy/tenant/remove:
+    post:
+      tags:
+        - Multitenancy Recipe
+      description: |
+        Delete a tenant.
+
+        Request must originate from public tenant.
+
+      operationId: deleteTenantPOST
+      parameters:
+        - $ref: '#/components/parameters/multitenancyRid'
+        - $ref: '#/components/parameters/api-key'
+        - $ref: '#/components/parameters/cdi-version'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - tenantId
+              properties:
+                tenantId:
+                  $ref: '#/components/schemas/tenantId'
```

Comment:
all remove APIs should have the tenantId / appId / cud as required

Reply:
My bad, it is already required in the existing changes. no changes are needed.